### DISCUSSION TITLE
Any way to interact with the browser?

### DISCUSSION THREAD

**Discussion by darabos** [nesbox/TIC-80#2133] [Q&A]
In PICO-8 you can use GPIO to communicate with JavaScript running in the page that is hosting the web build of your cart. (https://pico-8.fandom.com/wiki/GPIO#API) Is there anything like that in TIC-80?

Thank you!

**Comment by nesbox** [comment]
Unfortunately, nothing similar has been implemented yet, we only have opened issue #572
